components/atom/label/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+import type {FC, LabelHTMLAttributes} from 'react'
+
+import cx from 'classnames'
+
+import {CLASSNAME, FONT_SIZES, TYPES} from './settings'
+
+type FontSize = typeof FONT_SIZES[keyof typeof FONT_SIZES]
+type Type = typeof TYPES[keyof typeof TYPES]
+
+export interface AtomLabelProps extends Pick<LabelHTMLAttributes<HTMLLabelElement>, 'htmlFor' | 'onClick'> {
+  /**
+   * used as for attribute. Must be the same as the form element id
+   */
+  name: string
+  /**
+   * The label itself
+   */
+  text: string
+  /**
+   * Allows label to be displayed inline to de left
+   */
+  inline?: string
```

Comment:
I see in the code that this prop can be either left or right so maybe we can add some Type checking with these 2 values instead of using a string.

Reply:
Yep, it makes sense, it's not used for any other reason ;)